src/configurations/adknowledgeportal/routesConfig.ts

```diff
@@ -215,7 +216,21 @@ const routes: GenericRoute[] = [
       },
     ],
   },
-
+  {
+    // PORTALS-2028 (part 2): redirect /MODEL-ADstrains to /Explore/Experimental%20Models with query request
+    isNested: false,
+    to: 'MODEL-ADstrains',
+    hideRouteFromNavbar: true,
+    synapseConfigArray: [
+      {
+        name: 'Redirect',
+        props: {
+          from: 'MODEL-ADstrains',
+          to: { pathname: '/Explore/Experimental Models', search: `QueryWrapper0={"sql":"${encodeURI(experimentalModelsSql)}","limit":25,"offset":0,"selectedFacets":[{"concreteType":"org.sagebionetworks.repo.model.table.FacetColumnValuesRequest","columnName":"program","facetValues":["MODEL-AD"]}]}` },
```

Comment:
Would it be a good idea to co-locate the facet query with the `experimentalModelsSql` constant so that if the table updates, it's more likely that we remember to update both?